en/includes/guides/users/attributes/manage-attributes.md

```diff
@@ -69,28 +74,99 @@ To update the properties of a user attribute:
 
     <table>
        <tbody>
-          <tr>
-             <td><b>Attribute Display Name</b></td>
-             <td>Update the display name of the attribute that will show in user profiles.</td>
-          </tr>
-          <tr>
-               <td><b>Description</b></td>
-               <td>Update the description for the attribute.</td>
-          </tr>
+            <tr>
+                <td><b>Attribute Display Name</b></td>
+                <td>Update the display name of the attribute that will show in user profiles.</td>
+            </tr>
+            <tr>
+                <td><b>Description</b></td>
+                <td>Update the description for the attribute.</td>
+            </tr>
+            <tr>
+                <td><b>Regular expression</b></td>
+                <td>The value of the attribute will be validated against the regex pattern specified here.</td>
+            </tr>
+            <tr>
+                <td><b>Shared Profile Value Resolving Method</b></td>
+                <td>The method to determine the value of the attribute for shared user profiles in child organizations. 
+                <ul>
+                    <li><b>From Origin:</b> The value will be taken from the user origin organization's profile.</li>
+                    <li><b>From Shared Profile:</b> The value will be taken from the shared profile, if available.</li>
+                    <li><b>From First Found in Hierarchy:</b> The value will be determined from the first non-null profile value found in the organization hierarchy.</li>
+                </ul>
+                For {{ product_name }} defined attributes, the `Shared Profile Value Resolving Method` cannot be changed at the moment. However, for custom attributes, you can decide the appropriate resolving method.</td>
+            </tr>
+     </tbody>
+    </table>
+
+### Configure attribute profiles
+
+Configure attribute profiles for different flows in your organization.
```

Comment:
Let's add a bit more detail to explain the profiles.

Reply:
Added